spec.md

```diff
@@ -2622,10 +2622,10 @@ digit_hex = digit_dec
           | "a" | "b" | "c" | "d" | "e" | "f" ;
 
 (* An integer *)
-int = '"' , "b" , [ "-" ] , { digit_bin } , '"'
-    | '"' , "o" , [ "-" ] , { digit_oct } , '"'
-    | '"' , "h" , [ "-" ] , { digit_hex } , '"'
-    |             [ "-" ] , { digit_bin } ;
+int = '"' , "b" , [ "-" ] , { digit_bin }+ , '"'
+    | '"' , "o" , [ "-" ] , { digit_oct }+ , '"'
+    | '"' , "h" , [ "-" ] , { digit_hex }+ , '"'
+    |             [ "-" ] , { digit_dec }+ ;
```

Comment:
I'd prefer to use the pedantic `digit_dec , { digit_dec }` formulation as opposed to using non-standard, though obvious to anyone who's written a regex, `+`. This is only to avoid future confusion around what this means.

If there are problems fitting this on a page, feel free to use more preliminary definitions, e.g.,:

```
num_dec = digit_dec , { digit_dec } ;
```

I have been annoyed about this with EBNF that it doesn't have one-or-more. 🥲 

Ditto for other corrections to use `+`.

Reply:
Nice catch about `digit_dec` vs. `digit_bin`. 👍

Reply:
Okay, I'll explicitly list it then.